packages/caver-rpc/src/klay.js

```diff
@@ -2240,7 +2217,132 @@ class Klay {
+             * @instance
+             *
+             * @example
+             * // Use `caver.rpc.klay.getRewards`
+             * const result = await caver.rpc.klay.getRewards(0)
+             * const result = await caver.rpc.klay.getRewards('0x58482921af951cf42a069436ac9338de50fd963bdbea40e396f416f9ac96a08b')
+             *
+             * @param {string|number|BN|BigNumber} blockHashOrNumber The block hash or block number to query block header.
+             * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
+             * @return {Promise<Klay.Rewards>} An object includes reward distribution result.
+             */
+            new Method({
+                name: 'getRewards',
+                call: 'klay_getRewards',
+                params: 1,
+                inputFormatter: [formatters.inputBlockNumberFormatter],
+            }),
+            /**
+             * Provides the latest chain configuration.
+             * This is equivalent to chainConfigAt() with an empty parameter.
+             * To see the current information, please use {@link itemsAt}.
+             *
+             * @memberof Klay
+             * @method getChainConfig
+             * @instance
+             *
+             * @example
+             * const result = await caver.rpc.klay.getChainConfig()
+             *
+             * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
+             * @return {Promise<object>} The latest chain configuration
+             */
+            new Method({
+                name: 'getChainConfig',
+                call: 'klay_chainConfig',
+                params: 0,
+            }),
+            /**
+             * Returns the chain configuration at specific block.
+             * If the parameter is not set, it returns the chain configuration at the latest block.
+             * To see the current information, please use {@link itemsAt}.
+             *
+             * @memberof Klay
+             * @method getChainConfigAt
+             * @instance
+             *
+             * @example
+             * const result = await caver.rpc.klay.getChainConfigAt(0)
+             * const result = await caver.rpc.klay.getChainConfigAt('latest')
+             *
+             * @param {number|string} blockNumber A block number, or the hex number string to query chain configuration.
+             * @param {function} [callback] Optional callback, returns an error object as the first parameter and the result as the second.
+             * @return {Promise<object>} Chain configuration at the given block number.
+             */
+            new Method({
+                name: 'getChainConfigAt',
+                call: 'klay_chainConfigAt',
+                params: 1,
+            }),
+            /**
+             * Provides the address of the node that a user is using.
```

Comment:
What is the meaning of `user` in here? As far as I know, I think `an address of the operating node` is enough. How do you think?

Reply:
Yes, you are right. 
I couldn't check it carefully because I copied the existing sentence.

Reply:
Yes I pushed